irony-server.rst

```diff
@@ -0,0 +1,98 @@
+============
+irony-server
+============
+
+----------------------------------------
+The back-end server for Emacs irony-mode
+----------------------------------------
+
+:Author: Nicholas D Steeves <nsteeves@gmail.com>
+:Date: 2017-10-01
+:Version: 1.1.0
+:Manual section: 1
+
+
+Synopsis
+========
+
+irony-server [*options*] [*command*] [<*args*>]
+
+Description
+===========
+
+``irony-server`` provides the libclang interface to irony-mode. It uses a
+simple protocol based on S-expression. This server is written in C++ and
+requires the following packages to be installed on your system: CMake >=
+2.8.3, and libclang
+
+Options
+=======
+
+-v, --version
+
+ Show version and exit.
+
+-h, --help
+
+ Show more detailed command help.
+
+-i, --interactive
+
+-d, --debug
+
+--log-file PATH
+
+Commands
+========
+
+**candidates**
```

Comment:
I would write it like this, with the arguments on the same line as command name:

```rst
**candidates** PREFIX STYLE                                                                               
                                                                                                          
 Print completion candidates (require previous complete).                                                 
 STYLE is "exact", "case-insensitive" or "smart-case".                                                    
                                                                                                          
**complete** FILE LINE COL [-- [COMPILE_OPTIONS...]]                                                      
                                                                                                          
 Perform code completion at a given location.                                                             
```